ntbs-service/Pages/LegacyNotifications/Index.cshtml.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using ntbs_service.DataMigration;
+using ntbs_service.Models;
+using ntbs_service.Services;
+
+namespace ntbs_service.Pages.LegacyNotifications
+{
+    public class Index : PageModel
+    {
+        private readonly ILegacySearchService _legacySearchService;
+        private readonly INotificationImportService _notificationImportService;
+
+        [BindProperty] 
```

Comment:
Don't think we want this bound - this is not a value we populate from the request data!